### DISCUSSION TITLE
Invitation to Integrate Your GPU Pool into AIxBlock

### DISCUSSION THREAD

**Discussion by ghost** [akash-network/community#544] [Integrations]
Greetings,
We are [AIxBlock](https://aixblock.org/), a blockchain-based end-to-end ecosystem for AI that utilizes unused computing resources from BTC miners and idle global consumer GPUs. 

We're reaching out to explore a potential partnership with your team. We're keen to integrate your GPU pool into the AIxBlock ecosystem, enabling our community of AI builders to access your resources during idle periods.

One of the key advantages of partnering with AIxBlock is NO transaction fees for providers. Additionally, we provide access to a diverse global network of AI developers actively seeking affordable GPU resources.

For more information about the benefits of selling GPUs on AIxBlock, please refer to the details provided [here](https://drive.google.com/file/d/1bpOu1OCLo0ad27p-bHChwuVVVnKMtMk7/view?usp=sharing) 

We look forward to the opportunity to discuss this further and explore how we can collaborate effectively.

Best regards,

**Comment by ghost** [comment]
Hey there! Akash is a decentralized compute marketplace, providers come and provide compute to the Akash network

You are saying you would like to buy Akash resources for your network and clients??  

What does it take to integrate the Akash GPU pool into AlxBlock? 
How much would that cost to the Akash community? 
In what other ways will both project benefit?

**Reply by ghost** [reply]
Hi there,
Thanks for getting back to us. Please find our ans below:
1. Yes, we'd like to integrate it to our ecosystem so that our clients can rent it to train/run AI
2. No cost, we ll not charge any margin fee for GPU providers because our rev streams come from other AI services
3. Of course both projects will benefit, its a win-win one. We need more decentralized GPUs to be listed on our marketplace so that our AI builders have more options to choose from.
But we'd like to talk to someone who can decide some strategic partner because we think that there maybe a manual custom configuration we need to do to align with your tech and business model. For example, we r building on BNB chain, u r building on your own chain, so we may need do bridge chain, . etc. What we r proposing to other B2B GPU partners is that we ll consider them as a single entity, we ll build a custom dashboard for them, and all payment send to 1 single wallet, then that entity shall distribute the payment (after charging any margin on your end) to their GPU providers on their end

**Comment by brewsterdrinkwater** [comment]
This sounds like a very interesting opportunity. Thank you for taking the time to share with the Akash community. 

Do you have any direct asks of the Akash community at this time, or are you letting people know about AlxBlock? 

Have you been able to deploy any AI workloads on Akash compute resources, yet? What kind of compute resources work best for your customers?

Akash is open, so there should not be anything stopping you from integrating with Akash. There are other teams and organizations that are building custom workflows or leveraging [AkashJS](https://github.com/akash-network/akashjs) to integrate their tools with Akash compute resources. 

If you have specific questions about how to integrate, I would suggest joining a special interest group to give a quick demo to the community or what you have, and ways you think Akash can be integrated. I would suggest sig-clients or sig-support, where you can ask technical questions of the core team. Here is a calendar for the Akash special interest groups and working groups: https://akash.network/development/welcome/#calendar .

**Comment by ghost** [comment]
Hi,
1. We dont have any specific contact points from Akash to talk to yet, though when we twitted on X, your CEO/Founder and another tech team liked our idea and referred us to dicuss more on Discord and Github. 
2. We havent tried to deploy using Akash, but we read your documentation, and we also have some other members in our network used Akash to train AI and they said that Akash was super stable. 
Reg the kind of computes, for now, we support high performance GPUs from NVIDIA as a start, will open more soon. We checked your list of computes, sound like all of them are ideal. 
Our clients will need GPUs to build model scratch, we have our own comprehensive AI platform for them to build from ground up (from collecting data, label data, etc), then demo, deploy and even trade models.
3. Yes, we know u r opensource, but as u r building on your own chain, so we may need to have our tech team and yours to talk to each other and see how could we integrate you to our ecosystem, plus, we may need to also discuss a high level of business, for example: if we wanna consider all GPU providers from Akash as 1 entity, our clients pay to 1 single wallet, we build a cusom dashboard for all of your nodes, then you distribute the payment to your people (of course automatically) on your end? something like that.

**Reply by 31trainman** [reply]
I see an opportunity here to build something big but don't quite have all the details yet.

Join the Akash discord if your not already and I'll see what I can do to help. Here's the link https://discord.gg/akash

And my discord is 31trainman

**Reply by ghost** [reply]
hi, our team has just added you on Discord. Her name is Ha Dao. She'd like to talk more to you

**Reply by 31trainman** [reply]
Perfect I'll get a conversation going

**Reply by ghost** [reply]
Hi team,

We've just launched a private beta version of AIxBlock today. If you're interested, please join us and try it out: https://app.aixblock.io/user/signup

We're also looking into integrating Akash GPU into our compute marketplace
